### PR TITLE
keyboard

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-28-mechanical-keyboard.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+status: publish
+categories: [Writing, Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+# Mechanical keyboard
+
+I have just bought a mechanical custom made keyboard. It is really hard to use since it is a split keyboard without any label on any of the keys. Because of that my typing speed on it is abysmal. I'm so used to having a 'normal' keyboard that it completely threw me off. I'm also pretty fast at typing on a 'normal' keyboard without looking and I thought that it would translate in a decent performance here as well. Oh boys how wrong I was.
```
Comment (MsSlacker) [inline _posts/2019-11-28-mechanical-keyboard.md]: in - > into?

Comment (MsSlacker) [reply _posts/2019-11-28-mechanical-keyboard.md]: Boys - > boy

File: _posts/2019-11-28-mechanical-keyboard.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+I usually take around 45 seconds to type this much. Today it has taken me around 10 minutes.
+
+I have already written about [carpal tunnel syndrome] and how moving only the fingers instead of the whole hand helps on delaying it. I found myself moving my wrist too much so I thought that by using a split keyboard I could avoid having to move my hands too much. For now, as inexperienced as I am it is having the opposite effect, but I've been told that it takes a few weeks to get used to it and after those you are able to write really fast. On top of that it looks like you can have a way better hand posture after getting past the initial hassle of getting used to the new keyboard.
```
Comment (MsSlacker) [inline _posts/2019-11-28-mechanical-keyboard.md]: No link?

Comment (MsSlacker) [reply _posts/2019-11-28-mechanical-keyboard.md]: help s to delay

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-28-mechanical-keyboard.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-11-28-mechanical-keyboard.md]: should there be a comma after boy?

**Comment by valeIT** [comment]
Haven't published the CTS article yet so I guess better put this on hold until that is out or remove the link for now, will mark as WIP

**Review by MsSlacker — DISMISSED** [review]

**Comment by valeIT** [comment]
Merged them, is it ok?

File: _posts/2019-11-31-mechanical-keyboard.md [hunk 3]
```diff
@@ -11,16 +11,20 @@ image2:
...
+
 # Mechanical keyboard
 
-I have just bought a mechanical custom made keyboard. It is really hard to use since it is a split keyboard without any label on any of the keys. Because of that my typing speed on it is abysmal. I'm so used to having a 'normal' keyboard that it completely threw me off. I'm also pretty fast at typing on a 'normal' keyboard without looking and I thought that it would translate into a decent performance here as well. Oh boy how wrong I was.
+This brings up to today: I have just bought a mechanical custom made keyboard. It is really hard to use since it is a split keyboard without any label on any of the keys. Because of that my typing speed on it is abysmal. I'm so used to having a 'normal' keyboard that it completely threw me off. I'm also pretty fast at typing on a 'normal' keyboard without looking and I thought that it would translate into a decent performance here as well. Oh boy how wrong I was.
```
Comment (MsSlacker) [inline _posts/2019-11-31-mechanical-keyboard.md:26]: It really today anymore is it :P

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-31-mechanical-keyboard.md (hunk 3 above)
Comment (valeIT) [reply _posts/2019-11-31-mechanical-keyboard.md:26]: Well the publish date is 31/11 so the today is 'legit' imo xD

**Comment by valeIT** [comment]
You still need to approve this

**Pull request merged**